src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/GenerateEmailTransactionsFunction.cs

```diff
@@ -0,0 +1,43 @@
+﻿
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using SFA.DAS.ApprenticeCommitments.Jobs.Api;
+using SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure.Api.Requests;
+
+
+namespace SFA.DAS.ApprenticeFeedback.Jobs.Functions
+{
+    public class GenerateEmailTransactionsFunction
+    {
+        private readonly ILogger<GenerateEmailTransactionsFunction> _log;
+        private readonly IApprenticeFeedbackEmailTransactionApi _api;
+
+        public GenerateEmailTransactionsFunction(ILogger<GenerateEmailTransactionsFunction> log, IApprenticeFeedbackEmailTransactionApi api) //, IApprenticeFeedbackEmailTransactionService service)
```

Comment:
Pass in `IApprenticeFeedbackApi` and use the Outer Api that already exists, and remove the feedback transaction service in the constructor.

Reply:
done